packages/expansion-panel/src/components/expansion-panel/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {
+  FC,
+  isValidElement,
+  Children,
+  useState,
+  useEffect,
+  HTMLAttributes,
+  ComponentType
+} from 'react';
+
+import ExpansionPanelHead from '../expansion-panel-head';
+import ExpansionPanelBody from '../expansion-panel-body';
+
+import SC from './styled';
+
+export interface ExpansionIndicator {
+  expand: JSX.Element;
+  collapse: JSX.Element;
```

Comment:
Kanskje ReactNode (import fra`react`) passer bedre?

```suggestion
  expand: ReactNode;
  collapse: ReactNode;
```